dip-0011.md

```diff
@@ -80,10 +80,10 @@ The Blockchain Users document (DIP0005) aimed to lay out a foundational
 transformation of how end users would interact with one another. The underlying
 Blockchain User transactions represented a starting point for this interaction.
 Blockchain Users would have then been discoverable by querying their associated
-usernames. At a stark difference from DIP0005, Identities are decoupled from
+usernames. A stark difference from DIP0005, Identities are decoupled from
```

Comment:
Not sure here @thephez ?

Reply:
I'd tend towards the original. Probably would write something like "In stark contrast to DIP0005..." if I was writing it myself.

Reply:
https://github.com/dashpay/dips/pull/66/commits/095e6880cfdea9e4cb6c8100ef0682e1065e6123

Reply:
Okay other than the type :wink: Left a suggestion.